internal/app/machined/pkg/system/services/registry/store_darwin.go

```diff
@@ -0,0 +1,12 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Does this module get pulled into talosctl build? Then perhaps needs a Windows counterpart, else why does it need a Darwin implementation?